src/ert/scheduler/lsf_driver.py

```diff
@@ -330,7 +329,9 @@ async def poll(self) -> None:
                 continue
             current_jobids = list(self._jobs.keys())
             process = await asyncio.create_subprocess_exec(
-                self._bjobs_cmd,
+                str(self._bjobs_cmd),
+                "-noheader",
+                "-o \"jobid stat delimiter='^'\"",
```

Comment:
What is the advantage of using `^` instead of space?

Reply:
it doesn't really help with the current chosen columns, but it doesn't hurt either.
in general I find it nicer to use a delimiter that is unlikely to be used for anything in the text

Reply:
Would it be better to use triple quotes to avoid having to escape the inner quotes?

Reply:
it looked a bit out of place when I tried it earlier, but I can change it if you have strong feelings about it :)
both ways were accepted by ruff format

Reply:
Okay, lets leave it for now